src/components/DependencyManager.tsx

```diff
@@ -602,11 +605,13 @@ export const DependencyManagerButton = memo(() => {
                     />
                 </VStack>
             </Tooltip>
-            <DependencyManager
-                isOpen={isOpen}
-                onClose={onClose}
-                onPipListUpdate={setPipList}
-            />
+            <ExecutionProvider>
```

Comment:
I'm very sure this doesn't work. This execution context is not the same context as the `useContext(ExecutionContext)` you get `setIsBackendKilled` from. How you tested that this works?

Reply:
It doesn't work, I pinged you on discord about it

Reply:
@RunDevelopment it is now sorta working properly, but it still popped up with the warning, only it happened once instead of multiple times.